src/plugins/mer/merbuildconfiguration.cpp

```diff
@@ -0,0 +1,107 @@
+/****************************************************************************
+**
+** Copyright (C) 2019 Jolla Ltd.
+** Contact: http://jolla.com/
+**
+** This file is part of Qt Creator.
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 2.1 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL included in the
+** packaging of this file.  Please review the following information to
+** ensure the GNU Lesser General Public License version 2.1 requirements
+** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
+**
+** Other Usage
+**
+** Alternatively, this file may be used in accordance with the terms and
+** conditions contained in a signed written agreement between you and Digia.
+**
+****************************************************************************/
+
+#include "merbuildconfiguration.h"
+
+#include "merconnection.h"
+#include "merconstants.h"
+#include "merbuildsteps.h"
+#include "mersdk.h"
+#include "mersdkkitinformation.h"
+
+#include <projectexplorer/buildmanager.h>
+#include <projectexplorer/buildsteplist.h>
+#include <projectexplorer/projectexplorer.h>
+#include <projectexplorer/target.h>
+#include <qmakeprojectmanager/qmakeproject.h>
+#include <qmakeprojectmanager/qmakestep.h>
+
+using namespace ProjectExplorer;
+using namespace QmakeProjectManager;
+
+namespace {
+
+const char MER_USE_EXTRA_VARIABLES_KEY[] = "Mer.UseExtraVariablesIdentifier";
+const char MER_VARIABLES_CACHE_FILENAME[] = ".mb2/qmake_variables.cache";
+
+} // namespace anonymous
+
+namespace Mer {
+namespace Internal {
+
+MerBuildConfiguration::MerBuildConfiguration(Target *target, Core::Id id)
+    : QmakeBuildConfiguration(target, id)
+    , m_useExtraParserArguments(false)
```

Comment:
This is never changed in this commit, and after the later commit where the checkbox is added, MerSettings::isImportQmakeVariablesEnabled() should be used directly, without the need for a local copy of the state here. So I would remove this variable and the associated functions and assume "true" in this commit.

Reply:
Done.